src/Stomp/Network/Observer/Heartbeat/Emitter.php

```diff
@@ -0,0 +1,270 @@
+<?php
+/*
+ * This file is part of the Stomp package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Stomp\Network\Observer\Heartbeat;
```

Comment:
This is a single class Namspace.. should we instead strip heartbeat from the NS and name the class HeartbeatEmitter ?